### DISCUSSION TITLE
How to Implement VisualBrush Transform in Avalonia

### DISCUSSION THREAD

**Discussion by njdaoyehu** [AvaloniaUI/Avalonia#12138] [Q&A]
<Canvas
                    x:Name="Snapshot"
                    Cursor="{Binding CanLabel, Converter={StaticResource imageCursorConverter}}"
                    Width="1280"
                    Height="720"                
                    MouseLeftButtonDown="Snapshot_MouseLeftButtonDown"
                    MouseLeftButtonUp="Snapshot_MouseLeftButtonUp"
                    MouseMove="Snapshot_MouseMove"
                    MouseWheel="Snapshot_MouseWheel">
                <Canvas.Resources>
                    <TransformGroup x:Key="ImageTransformResource">
                        <ScaleTransform />
                        <TranslateTransform />
                    </TransformGroup>
                </Canvas.Resources>
                <Canvas.Background>
                    **_<VisualBrush Transform="{StaticResource ImageTransformResource}" Stretch="Uniform">_**
                        <VisualBrush.Visual>
                            <Canvas x:Name="ImageContainer" Width="1280" Height="720">
                                <Canvas.Background>
                                    <ImageBrush ImageSource="{Binding address}" Stretch="Fill" />
                                </Canvas.Background>
                            </Canvas>
                        </VisualBrush.Visual>
                    </VisualBrush>
                </Canvas.Background>
            </Canvas>

**Comment by timunie** [comment]
works for me using Avalonia 11 and `DynamicResource`
